Source/UI/RemoveScavModeButtonPatch.cs

```diff
@@ -1,7 +1,6 @@
 ﻿using EFT.UI;
 using SIT.Tarkov.Core;
 using System.Reflection;
-using UnityEngine;
```

Comment:
Could you put this back as we are gonna use it to disable the Scav Button, Thank you